### DISCUSSION TITLE
How to boot Vinix

### DISCUSSION THREAD

**Discussion by vajrabisj** [vlang/v#15295] [Installation Issues]
I downloaded iso file, how to boot it on my Mac m1?

**Comment by Wajinn** [comment]
Have you tried [unetbootin](https://unetbootin.github.io/)?  Lots of people make bootable USBs from ISOs.

**Reply by vajrabisj** [reply]
Thanks, I will try it

**Reply by Wajinn** [reply]
Let people know how it goes.  Also, another possibility for creating a bootable USB from an ISO, is [etcher](https://github.com/balena-io/etcher).
